drivers/net/slip.c

```diff
@@ -109,6 +109,11 @@
 
 #define SLIP_WDDELAY   (1*1000000)
 
+/* This is a helper pointer for accessing the contents of the ip header */
+
+#define IPv4HDR        ((FAR struct ipv4_hdr_s *)priv->dev.d_buf)
```

Comment:
Maybe similar change is need in `tun.c`. @anchao could you please take a look?

Reply:
@wengzhe let's switch slip.c to iob like tun.c too.

Reply:
@pkarashchenko @xiaoxiang781216 maybe PPPD is not working because it uses a TUN interface, what do you think?

Reply:
@wengzhe already prepare a patch to convert tun.c using iob offload, he will do the same thing for slip.c too. Since we are in Spring Festival holiday, the patch will be provided early next week.

Reply:
@michi-jung could you try whether this patch fix your issue: https://github.com/apache/nuttx/pull/8344?